main.tf

```diff
@@ -1,27 +1,23 @@
+locals {
+  enabled = module.this.enabled
```

Comment:
Why create a local alias instead of just using `module.this.enabled` considering this doesn't have any other logic?

Reply:
In case of we want add some logic for this we will just modify local variable one time instead of replacing multiple conditions.